node/task.ts

```diff
@@ -614,20 +597,38 @@ export const checkPath = im._checkPath;
  */
 export function cd(path: string): void {
     if (path) {
-        shell.cd(path);
-        _checkShell('cd');
+        let errMsg: string = '';
+        if (!fs.existsSync(path)) {
```

Comment:
iirc the node pattern is do not call `existSync`, just `statSync` and catch ENOENT

Reply:
also i think the statSync function is in the task lib, it may have a localized error already for the ENOENT case